patches/web_contents_impl.patch

```diff
@@ -1,8 +1,24 @@
 diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
-index b5b0b78..aac6876 100644
+index b5b0b78..1c42b6c 100644
 --- a/content/browser/web_contents/web_contents_impl.cc
 +++ b/content/browser/web_contents/web_contents_impl.cc
-@@ -4353,6 +4353,7 @@ void WebContentsImpl::RendererUnresponsive(
+@@ -1820,10 +1820,14 @@ void WebContentsImpl::CreateNewWindow(
+       static_cast<SessionStorageNamespaceImpl*>(session_storage_namespace);
+   CHECK(session_storage_namespace_impl->IsFromContext(dom_storage_context));
+ 
++  WindowContainerType window_container_type = WINDOW_CONTAINER_TYPE_NORMAL;
++  if (params.disposition == NEW_BACKGROUND_TAB)
```

Comment:
`frame_name` to CanCreateWindow looks like something that can likely be upstreamed to Chrome since it appears to be a legitimate bug affecting popup handling. In Chrome it should also be added to `BlockedWindowParams`